CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to `Laravel settings` will be documented in this file
 
+## 2.0.0 - 2024-05-14
```

Comment:
I'm still considering if this should be a major release or not. Although we are dropping support for Laravel 8 & 9 your composer.json file will ensure the correct version is being used. 

I think a 1.5.0 release could cover the case.

Reply:
As long as composer indeed does ensure correct versions, both options are fine for me.

Reply:
Neem voor de zekerheid maar 2.0.0, ik ben vrij zeker dat 1.5.0 ook gaat werken maar niet 100%